assets/src/application/ui/calendars/dateDisplay/MediumCalendarDate/index.tsx

```diff
@@ -46,11 +46,12 @@ export const MediumCalendarDate: React.FC<MediumCalendarDateProps> = ({
 			{headerText && <div className='ee-medium-calendar-date__header'>{headerText}</div>}
 			<div className='ee-medium-calendar-date'>
 				<div className='ee-mcd__weekday'>{format(dateObject, WEEKDAY_ONLY_FULL_FORMAT)}</div>
-				<div className='ee-mcd__month-day'>
+				<div className='ee-mcd__month-day-year'>
 					<span className='ee-mcd__month'>{format(dateObject, MONTH_ONLY_LONG_FORMAT)}</span>
 					<span className='ee-mcd__day'>{format(dateObject, DAY_ONLY_SHORT_FORMAT)}</span>
+					<span className='ee-mcd__separator'>/</span>
+					<span className='ee-mcd__year'>{format(dateObject, YEAR_ONLY_SHORT_FORMAT)}</span>
 				</div>
-				<div className='ee-mcd__year'>{format(dateObject, YEAR_ONLY_LONG_FORMAT)}</div>
```

Comment:
tightened this up to make more room for the edit date range button in the sidebar